### ISSUE TITLE
Support for updates

### ISSUE THREAD

**Issue by bluestreak01** [questdb/roadmap#7]
## Summary

We aim to support updates in use-cases where heavy data-acquisition is still the king, but occasional data corrections are still required. Users will be able leverage `update` SQL syntax in read-consistent context. `delete` operations are out of scope of this issue.

## Expected outcome

The following syntax will be supported

```ebnf
[ WITH with_query [, ...] ]
UPDATE table_name [ [ AS ] alias ]
    SET { column_name = { expression } |
          ( column_name [, ...] ) = ( { expression } [, ...] ) |
          ( column_name [, ...] ) = ( sub-SELECT )
        } [, ...]
    [ FROM from_item [, ...] ]
    [ WHERE condition ]
```

This issue depends on #9

**Comment by Shivansh-Mittal** [comment]
Did update query support in questdb implemented ?
Can i use update query now in questdb ?
Kindly help by answering if you know.

**Comment by Shivansh-Mittal** [comment]
Did update query support in questdb implemented ?
Can i use update query now in questdb ?
Kindly help by answering if you know.
